.github/workflows/packages.yml

```diff
@@ -170,7 +170,7 @@ jobs:
         done
     - name: Free additional disk space (if necessary)
       run: |
-        if grep -Eq '^(chromium|code-server|code-oss|electron-headers-.*|rustc-nightly|python-polars)$' ./built_tur_packages.txt; then
+        if grep -Eq "^($(paste -s -d '|' ./big-pkgs.list))$" ./built_tur_packages.txt; then
           sudo apt purge -yq $(dpkg -l | grep '^ii' | awk '{ print $2 }' | grep -P '(cabal-|dotnet-|ghc-|libmono|php|aspnetcore)') \
```

Comment:
Should update these packages if the host VM bumps to Ubuntu 24.04.

Reply:
If you are afraid that some package will not be found, this is not true. In termux-packages when the ubuntu version was updated, the packages that are removed to free up space were not updated. You can see it [here](https://github.com/termux/termux-packages/blob/master/.github/workflows/packages.yml#L203).

Or do you want to add other packages to free up more space?

Reply:
IIRC it will not work, see https://github.com/termux/termux-packages/commit/c392e1bf75eda1b01075b5e1ac3e1de3340c3a99

Reply:
> IIRC it will not work, see https://github.com/termux/termux-packages/commit/c392e1bf75eda1b01075b5e1ac3e1de3340c3a99

There (in the termux-packages repository) was a problem with removing the `google-cloud-cli` package and the `powershell` package. As far as I know, the yml scripts that build packages in tur do not remove these packages. The exception is `package_on_device.yml` and `package_update.yml`, since the first one uses your own VM, and the second one does not have updated ubuntu (i.e. it uses version 22.04).

> Better to copy them and keep up to date with the main repo.

That is, you want to add other packages that will be removed to free up space?

Reply:
Yes. It is better to sync with the main repository.